.circleci/test-deploy.yml

```diff
@@ -105,7 +105,8 @@ jobs:
           configure_git: false
           image_context: ./fixtures/docker
           image_name: busybox
-          image_tag: ${CIRCLE_SHA1}
```

Comment:
@pieeri we will see in docker hub both tags `latest` and `test` pushed at the same time and we won't know for sure if `tag_only` worked

Reply:
@zyassine Maybe the images will be successfully pushed into `Dockerhub` ; however all steps could be introspected in CircleCI.

Especially, as detailed in the snippet above,
- one step will show an image with a specific SHA, pushed with the latest and CircleCI tag.
- one step will show a image with a similar SHA pushed under the test tag (hopefully Docker layers will be cached onto the machine) 

It will be very clear if any of the steps has failed or not.